README - INSTALL INSTRUCTIONS/QBCore/README.md

```diff
@@ -226,7 +226,7 @@ Delete `qb-apartments/config.lua` references in `qb-garages` fxmanifest.lua.
 
 # 
 
-### For qb-garages 2.0 version: 
+### For qb-garages 2.0 version: https://github.com/qbcore-framework/qb-garages
```

Comment:
It would be a good idea to refer to the exact release. Otherwise, people will complain when QB does some backward incompatible stuff that breaks the script.

Reply:
2.0 as they have it in their Github

Reply:
Ew, didn't release they are not working with releases.